pkg/api/common.go

```diff
@@ -26,10 +29,74 @@ var (
 	maxTimeFormatted = maxTime.Format(time.RFC3339Nano)
 )
 
+func ParseFlags(cfg *Config) *Config {
```

Comment:
nit: If we are accepting `Config` as reference, then why does `ParseFlags` need to return it? We can simply getrid of the return as its redundant here.

Reply:
Replicated the same convention from other places, I don't see the problem there.

Reply:
The main point here is of returning an object which wasnt used. Name isnt much of an issue.

Reply:
No, the main point is that you are complaining about something which already present in the codebase. 
I prefer to adhere to the codebase standards. 
If you have problems with this, please open a separate PR details why its worthwhile to change all these instances across the whole project.

Please keep in line to codebase idioms.

Examples:
https://github.com/timescale/promscale/blob/master/pkg/log/log.go#L36
https://github.com/timescale/promscale/blob/master/pkg/pgclient/config.go#L35
https://github.com/timescale/promscale/blob/master/pkg/runner/runner.go#L61